txscript/engine.go

```diff
@@ -335,8 +335,6 @@ func (vm *Engine) isBranchExecuting() bool {
 // conditional).
 func isOpcodeDisabled(opcode byte) bool {
 	switch opcode {
-	case OP_CAT:
-		return true
```

Comment:
Just curious. Does removing this case still throw some error when CAT (0x7e) is being used in a segwitv0 or legacy context?

Reply:
Yes, if opcode is being executed without a taproot context, it will fall back to old behavior: https://github.com/btcsuite/btcd/pull/2095/commits/9541657983b8a54e3225e450bcd961e2c29eb29b#diff-27954f8a5853761d7c36b3084d7eb53d7c9a6788eac76b4b20dc573f723a8296R1954